Sources/Subscription/FeatureFlags/FeatureFlaggerMapping.swift

```diff
@@ -18,18 +18,16 @@
 
 import Foundation
 
-public enum SubscriptionFeatureName: String, CaseIterable {
-    case privateBrowsing = "private-browsing"
-    case privateSearch = "private-search"
-    case emailProtection = "email-protection"
-    case appTrackingProtection = "app-tracking-protection"
-    case vpn = "vpn"
-    case personalInformationRemoval = "personal-information-removal"
-    case identityTheftRestoration = "identity-theft-restoration"
-}
+open class FeatureFlaggerMapping<Feature> {
```

Comment:
It's a shame we have to add this layer of indirection, but I see why it's needed - reasonable workaround :+1: